.github/workflows/update_robottelo_image.yml

```diff
@@ -0,0 +1,60 @@
+# Update robottelo image on quay.
+name: update_robottelo_image
+
+on:
+  push:
+    branches:
+      - master
+      - 6.*.z
+  repository_dispatch:
+    types:
+      - airgun-pr-merged
+      - nailgun-pr-merged
+
+jobs:
+  robottelo_container:
+    name: Update Robottelo container image on Quay.
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Get branch info
+        id: repo_branch
+        run: |
+          BRANCH="${{ github.event.client_payload.branch }}"
+          if [ -z "$BRANCH" ]; then
+            BRANCH="${GITHUB_REF##*/}"
+          fi
+          echo "REPO_BRANCH=$BRANCH" >> $GITHUB_OUTPUT
```

Comment:
In the case of a push event, this above line will not be executed as evaluating this expression would fail `BRANCH="${{ github.event.client_payload.branch }}"` nothing would run. this is my extensive guess can you confirm once? I'm okay it that is working then